content/ru/observability.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Observability
```

Comment:
I think we should have the translated term here, the same way we did for [Cloud Native tech](https://glossary.cncf.io/ru/cloud-native-tech/) or [CI](https://glossary.cncf.io/ru/continuous-integration/). Otherwise, we might end up with some terms being in Russian and others in English, which breaks the consistency. WDYT?
```suggestion
title: Наблюдаемость
```